packages/database/.babelrc.js

```diff
@@ -0,0 +1,38 @@
+const env = {
+  test: {
+    plugins: ['istanbul'],
+  },
+};
+
+const includedDateOffset = 90 * 24 * 60 * 60 * 1000; // include migrations up to 90 days old
+const includedMigrationsDate = new Date().setTime(Date.now() - includedDateOffset);
+const checkMigrationWithinIncludedRange = function (migrationName) {
+  const yearPart = migrationName.substring(0, 4);
+  const monthPart = migrationName.substring(4, 6);
+  const dayPart = migrationName.substring(6, 8);
+  const migrationDate = new Date(`${yearPart}-${monthPart}-${dayPart}`);
+  return migrationDate < includedMigrationsDate;
```

Comment:
This works but the function name is a bit misleading, I suggest either renaming it to `checkMigrationOutsideIncludedRange`, or reversing the operator to `>` and adding a `!` where it's called.

Reply:
Oh yep good call, will rename